net/crowdsec-firewall-bouncer/Makefile

```
@@ -0,0 +1,122 @@
+#
+# Copyright (C) 2021 Gerald Kerma
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=crowdsec-firewall-bouncer
+PKG_VERSION:=0.0.13
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/crowdsecurity/cs-firewall-bouncer
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_DATE:=20210705
+PKG_MIRROR_HASH:=7490a99288246bbc883384c47b441bb2f3867724a9f1a0e94d352fe36d47c220
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+CSFB_BUILD_VERSION?=$(PKG_SOURCE_VERSION)
+CSFB_BUILD_GOVERSION=$(shell go version | cut -d " " -f3 | sed -r 's/[go]+//g')
+CSFB_BUILD_TIMESTAMP=$(shell date +%F"_"%T)
```

```suggestion
CSFB_BUILD_TIMESTAMP=$(SOURCE_DATE_EPOCH)
```

`$(SOURCE_DATE_EPOCH)` is a Unix timestamp set by the buildroot/sdk to support reproducible builds, i.e. if you compile the same package with the same sdk at different times, the binaries you get should (ideally) be identical.

---

Good to know !
Applied...
Thanks